sts-mock/openapi/sts-mock-spec.yaml

```diff
@@ -96,7 +96,7 @@ paths:
     post:
       operationId: postToken
       summary: Obtain a service token.
-      description: Exchange a Mock STS-issued access token for a service token.
+      description: Exchange an access token for a mock service token.
```

Comment:
Should this say exchange a mock subject token for a mock service token?

Reply:
It's referred to as STS access token in the documentation. Can we have this instead?
`description: Exchange an STS access token (subject token) for a mock service token.`

Reply:
Absolutely - just wanted to get subject token _somewhere_ in there